sys/include/net/gcoap.h

```diff
@@ -626,8 +626,16 @@ extern "C" {
 #define GCOAP_DTLS_EXTRA_STACKSIZE  (0)
 #endif
 
+#if IS_USED(MODULE_COAPFILESERVER)
+#include "vfs.h"
+#define GCOAP_VFS_EXTRA_STACKSIZE   (VFS_DIR_BUFFER_SIZE + VFS_FILE_BUFFER_SIZE)
```

Comment:
Doxygen camplains that this is not documented. Maybe that disappears, if the above "@brief" is changed to "@name"